### DISCUSSION TITLE
Modify dropdown menu example

### DISCUSSION THREAD

**Discussion by cristiano-linvix** [floating-ui/floating-ui#3057] [Q&A]
Hello. I hope everyone is doing well.

I'm trying to change the dropdown example so that the root menu is not a button.
I want the floating dropdown menu to remain exactly as it is, but I don't want the root dropdown as a visible button on the screen.
I want to attach this dropdown menu to several possible elements, according to my needs.

I want to create a rootmenu that receives a forwardref so that I can open the menu programmatically in any component by passing the reference component.

like this:
```ts
const open = useCallback((refElement: HTMLElement) => {
refs.setReference(refElement);
setIsOpen(true);
}, []);
```

However, I want to keep the multi-level menu functionality as the example shows.

I don't have a good grasp of floating UI yet, I would really appreciate it if someone could help me or give me a way to do it.

[Link for official example](https://codesandbox.io/p/sandbox/admiring-lamport-5wt3yg?file=%2Fsrc%2FDropdownMenu.tsx%3A1%2C1-263%2C1)

A final structure, like this
```ts
<RootMenu ref={refMenu}>
  <MenuItem label="Undo" onClick= {()=> console.log("Undo")} />
  <MenuItem label="Redo" disabled />
  <MenuItem label="Cut" />
  
  <SubMenu label="Copy as">
    <MenuItem label="Text" />
    <MenuItem label="Video" />
    <SubMenu label="Image">
      <MenuItem label=".png" />
      <MenuItem label=".jpg" />
      <MenuItem label=".svg" />
      <MenuItem label=".gif" />
    </SubMenu>
    <MenuItem label="Audio" />
  </SubMenu>

  <SubMenu label="Share">
    <MenuItem label="Mail" />
    <MenuItem label="Instagram" />
  </SubMenu>
</RootMenu>
```

**Comment by cristiano-linvix** [comment]
Hello @atomiks, can you help-me please?

**Comment by cristiano-linvix** [comment]
Sorry, @atomiks , can you help-me please?

**Comment by cristiano-linvix** [comment]
@atomiks, can you help-me please?

**Reply by atomiks** [reply]
Hi, I recommend using a preconfigured library instead like Radix's dropdown menu. I haven't responded because I'm busy and can't help with issues that aren't basic to look at or debug.

**Reply by cristiano-linvix** [reply]
Thanks for the answer!

I understand the suggestion, but I don't like to attach too many dependencies to my projects because of the bundle.

Even so, I appreciate your answer.

If you could take a look, I'd appreciate it!

Have a great day!
